src/core/src/op/tile.cpp

```diff
@@ -88,16 +62,13 @@ bool op::v0::Tile::evaluate_tile(const HostTensorVector& outputs, const HostTens
     ov::Shape data_shape = data->get_shape();
     auto data_rank = data_shape.size();
     auto output_rank = std::max(data_rank, repeats_rank);
-
-    // expand data shape and repeats to output rank
-    data_shape.insert(data_shape.begin(), output_rank - data_rank, 1);
     repeats_val.insert(repeats_val.begin(), output_rank - repeats_rank, 1);
```

Comment:
You don't need to modify input shape before shape_inference. You can just use axis->get_shape()
Also, you can fill the `constant_data` map (last parameter for shape_infer) with axis_value
It can help to fix failed tests